linkerd/trace-context/src/service.rs

```diff
@@ -35,20 +36,35 @@ impl<K: Clone, S> TraceContext<K, S> {
     }
 
     fn request_labels<B>(req: &http::Request<B>) -> HashMap<&'static str, String> {
-        let mut labels = HashMap::with_capacity(5);
-        labels.insert("http.method", format!("{}", req.method()));
-        let path = req
-            .uri()
-            .path_and_query()
-            .map(|pq| pq.as_str().to_owned())
-            .unwrap_or_default();
-        labels.insert("http.path", path);
-        if let Some(authority) = req.uri().authority() {
-            labels.insert("http.authority", authority.as_str().to_string());
+        let mut labels = HashMap::with_capacity(6);
```

Comment:
nit: it looks like there are 7 entries:
`url.scheme`, `url.path`, `url.query`, `server.address`, `server.port`, `http.request.method`, and `http.response.status_code`.

i'll call this a nit only because the capacity of `5` that was there before this PR has the same issue.